apache/trafficcontrol
# Validating pattern for POST regex call in DS

## What does this PR (Pull Request) do?


- [x] This PR fixes #4981  


## Which Traffic Control components are affected by this PR?


- Traffic Ops

## What is the best way to verify this PR?


1. Run Traffic_Ops locally
2. Run POST call to https://<TO's-url:port>/api/3.0/deliveryservices/ds-id/regexes/ with a JSON body. For eg: 
```
{
    "pattern": "",
    "type": 33,
    "setNumber": 17
}
```

Note: You should the following response
```
   {"alerts":
       [{
           "text":"'pattern: cannot be blank.",
           "level":"error"}
       ]}
   }
```

## If this is a bug fix, what versions of Traffic Control are affected?


- master (a169205)
- 4.1.0

## The following criteria are ALL met by this PR




## Additional Information


Documentation has a correct example.
